cmd/migrate/main.go

```diff
@@ -13,8 +13,6 @@ func main() {
 
 	db := database.GetDB()
 
-	db.Exec("CREATE DATABASE faladev;")
```

Comment:
Talvez podemos manter essa linha usando um workaround para o postgres para criar o banco somente se ele não existir, pra melhorar a DevEx.

```sql
SELECT 'CREATE DATABASE mydb'
WHERE NOT EXISTS (SELECT FROM pg_database WHERE datname = 'mydb')\gexec
```

Fonte: https://stackoverflow.com/questions/18389124/simulate-create-database-if-not-exists-for-postgresql

Reply:
@tenlisboa a ideia é subir tudo pelo docker-compose. O docker compose já cria a base.